packages/manager/modules/bm-server-components/src/netboot/template.html

```diff
@@ -55,12 +76,12 @@
                             class="form-control"
                             id="rescue"
                             name="rescue"
-                            data-ng-options="rescue as rescue.kernel for rescue in $ctrl.netboots.rescue"
+                            data-ng-options="rescue as rescue.kernel + ' - ' + rescue.description for rescue in $ctrl.netboots.rescue track by rescue.kernel"
```

Comment:
are you sure the kernel attrib will unique ?
if not, consider using $index instead.

Reply:
@mohammed-zahaf  yes because it depends on the filenames from here: https://stash.ovh.net/projects/DEDICATED/repos/ipxe-scripts/browse/ipxe